interface/menu.dm

```diff
@@ -76,6 +76,23 @@ GLOBAL_LIST_EMPTY(menulist)
 /datum/verbs/menu/Icon/Size
 	checkbox = CHECKBOX_GROUP
 	default = /datum/verbs/menu/Icon/Size/verb/iconstretchtofit
+/datum/verbs/menu/Icon/Size/Load_checked(client/C)
+	var/atom/verb/verbpath = Get_checked(C)
+	if (!verbpath || !(verbpath in typesof("[type]/verb")))
```

Comment:
[2nd item](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#enforced-not-enforced)